lib/core/test/unit/Cardano/Pool/DB/Arbitrary.hs

```diff
@@ -164,9 +165,16 @@ instance Arbitrary PoolOwner where
         byte <- elements ['0'..'8']
         return $ PoolOwner $ B8.pack (replicate 32 byte)
 
+instance Arbitrary PoolFlag where
+    arbitrary = elements
+        [ Delisted
+        , NoPoolFlag
+        ]
```

Comment:
Hi @hasufell

I think this one was marked "resolved" without a resolution. :smile_cat: 

IMO, it's much better to use `arbitraryBoundedEnum` here.

If we add more constructors to `PoolFlag` (for whatever reason), this `Arbitrary` instance will continue to compile, but it will now generate only a subset of the constructors. :-1: 

Though if we use `arbitraryBoundedEnum`, it'll continue generating all constructors even if we add more in future.

(It's also fairly standard throughout our testing code to use `arbitraryBoundedEnum`.)